src/service/viewer-impl.js

```diff
@@ -90,9 +91,6 @@ export class Viewer {
     /** @const {!Window} */
     this.win = ampdoc.win;
 
-    /** @private @const {boolean} */
-    this.isIframed_ = (this.win.parent && this.win.parent != this.win);
```

Comment:
if we really want to cache it, we can cache globally inside `dom.js`? But I don't see such a need either.